src/main/resources/fix_mod_consortia_table.sql

```diff
@@ -0,0 +1,20 @@
+DO $$
+DECLARE
+    target_tenant TEXT:= 'consortium';
+    source_tenant TEXT:= 'college';
+    record RECORD;
```

Comment:
please add readme how to use this fuction
ANd I will provide also example how to use API part

Reply:
added explanation